doc/workshop/manipulation/clipping/clipping.json

```diff
@@ -9,8 +9,8 @@
             "type": "filters.overlay"
         },
         {
-            "expression": "Classification == 6",
-            "type": "filters.expression"
+            "limits": "Classification[6:6]",
+            "type": "filters.range"
```

Comment:
If memory serves, a significant chunk of the recent changes was going from `filters.range` to `filters.expression`, this change would undo one of those changes, is there a particular reason for this change?

(I could also be remembering wrong)